### PR TITLE
Update/revisions button shouldn't show with zero revisions

### PR DESCRIPTION
**Pull request by adamsilverstein**
Fixes https://github.com/WordPress/gutenberg/issues/1108

### CODE REVIEW

File: editor/sidebar/last-revision/index.js
Comment (youknowriad) [inline editor/sidebar/last-revision/index.js]: Could we move this under the check and drop the ternary? 
There's also another ternary we could drop for the `href` prop.

File: editor/sidebar/last-revision/index.js
Comment (aduth) [inline editor/sidebar/last-revision/index.js]: For an empty state, `null` might be the more appropriate return value, but this works just as well.

File: editor/sidebar/last-revision/index.js
Comment (aduth) [inline editor/sidebar/last-revision/index.js]: Would it ever be negative? Maybe `! revisions.length` ? Or at least `revisions.length === 0` to more accurately describe what we're anticipating.

File: editor/sidebar/last-revision/index.js
Comment (adamsilverstein) [inline editor/sidebar/last-revision/index.js]: I think either works and wasn't sure what the standard approach you use for the project. I'll see if I can find  other examples.

File: editor/sidebar/last-revision/index.js
Comment (adamsilverstein) [inline editor/sidebar/last-revision/index.js]: Wasn't sure if value could be non 0, for example maybe it would be false if revisions are disabled entirely.  I'll do some testing and see if we can improve this.

File: editor/sidebar/last-revision/index.js
Comment (adamsilverstein) [inline editor/sidebar/last-revision/index.js]: Good suggestion. Was trying to minimize the change set.  I will take another look.

Comment (adamsilverstein) [inline editor/sidebar/last-revision/index.js]: Fixed this in the latest version

**Review by youknowriad — APPROVED** [review]
👍 Great.

File: editor/sidebar/last-revision/index.js
Comment (adamsilverstein) [inline editor/sidebar/last-revision/index.js]: fixed this, checking for ! revisions now

File: editor/sidebar/last-revision/index.js
Comment (adamsilverstein) [inline editor/sidebar/last-revision/index.js]: switched this to `return null` as this is in fact what the React docs recommend, see https://facebook.github.io/react/docs/conditional-rendering.html

**Comment by adamsilverstein** [comment]
@mtias or @aduth happy to merge this if you give me the capability

**Comment by youknowriad** [comment]
@adamsilverstein I added you to the repo. It should be good now

**Comment by adamsilverstein** [comment]
Thanks @youknowriad.

**Comment by adamsilverstein** [comment]
Hmm, @youknowriad seems like I still can't merge :) not certain why but says I don't have write access.

**Comment by adamsilverstein** [comment]
oh, I had to accept the invitation :) sorry for the noise, i'm in now @youknowriad. Thanks!

**Comment by youknowriad** [comment]
Small Tip: I don't know if it's important for Gutenberg, but may be better to use `git rebase` instead of `git merge` to avoid all these merge commits

**Comment by mtias** [comment]
Yes, please, let's use `git rebase` and keep history clean.

**Comment by adamsilverstein** [comment]
@youknowriad ok, sure... are you referring to merging master back into my branch and you would rather i rebase the changes on top of current master?

**Pull request merged**
